website/docs/addon_applications_admin.md

```diff
@@ -15,7 +15,10 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 {versions.Applications_Badge}
 </ReactMarkdown>
 
-## Introduction
+
+## Applications Addon Features
+
+### Applications, Tools and Filters
 
 The Applications addon streamlines the setup of applications and tools across your studio. It empowers you to set executable paths and startup environments, with support for various operating systems including Windows, Linux, and macOS.
```

Comment:
empowers is the wrong word. Use "allows" instead.

Reply:
Thank you. 
https://github.com/ynput/ayon-documentation/pull/287/commits/069be35feeac7fabad542741b1bfa6d7643d8195